preprocessors/action.schema.json

```
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/action-recognition.schema.json",
+    "type": "object",
+    "title": "Human Action Recognition",
+    "description": "Preprocessor to detect human action in an image. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.actionRecognition'. Up to one action is assigned to each person detected in the image by object detection preprocessor.",
+    "properties":{
+        "actions":{
+            "type":"array",
+            "description": "Contains a list of actions detected by the preprocessor 'ca.mcgill.a11y.image.preprocessor.actionRecognition' or empty list if no actions are found. Each action is associated with a person detected by the preprocessor 'ca.mcgill.a11y.image.preprocessor.objectDetection' and identified by 'personID'.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "personID":{
+                        "description": "The ID of a person. The ID is generated by the preprocessor 'ca.mcgill.a11y.image.preprocessor.objectDetection' and is reused in this preprocessor",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "action":{
+                        "description": "Action label: Labels are one to three words long with words separated by an underscore. Action label contains all lowercase letters and first word is a verb ending in 'ing'.",
+                        "type": "string",
+                        "pattern": "[a-z]+ing(_[a-z]+)*"
```

Missed the presence of `push_ups` and `baby_crawling` which break with this pattern...making it more generic and ensuring it matches the whole line.
```suggestion
                        "pattern": "^[a-z]+(_[a-z]+)*$"
```